chia/full_node/full_node.py

```diff
@@ -1377,12 +1377,12 @@ async def ingest_blocks(
                     raise ValueError(f"Failed to validate block batch {start_height} to {end_height}: {err}")
                 if end_height - block_rate_height > 100:
                     now = time.monotonic()
-                    block_rate = int((end_height - block_rate_height) // (now - block_rate_time))
+                    block_rate = (end_height - block_rate_height) / (now - block_rate_time)
                     block_rate_time = now
                     block_rate_height = end_height
 
                 self.log.info(
-                    f"Added blocks {start_height} to {end_height} ({block_rate} blocks/s) (from: {peer.peer_info.ip})"
+                    f"Added blocks {start_height} to {end_height} ({block_rate:.3g} blocks/s) (from: {peer.peer_info.ip})"
```

Comment:
Do we want exponential notation here?

example:
```python-console
>>> f"{123456.7890123:.3g}"
'1.23e+05'
```

Reply:
I did intentionally choose `.g` over `.f`, yes. With `.3g` you get non-exponential notation for all the likely real-world cases.

```python
>>> for i in (10000, 9999, 1000, 999, 100, 99, 10, 9, 1, 0.9, 0.1, 0.01, 0.001, 0.0001, 0.00009):
...     print(f"{i:.3g}")
...
1e+04
1e+04
1e+03
999
100
99
10
9
1
0.9
0.1
0.01
0.001
0.0001
9e-05
```